resample_states_verification/NOTES.md

```diff
@@ -0,0 +1,58 @@
+
+Testing of changes to use mock GET_PITCH commands
+=================================================
+
+Used the timelines code to make a new test database and cmd states for 2010.  This made
```

Comment:
Do the two lines below accomplish the "Used the timelines code to make..." action?  I am assuming so but it isn't entirely clear.  You might say "Created a new test database and cmd states for 2010 using the commands below from the timelines repository."